src/services/transactions.service.ts

```diff
@@ -6,11 +6,11 @@ import { ValueNormalizer } from '../utils/value-normalizer.utils';
 import { TagsService } from './tags.service';
 import { TransactionMetaDto } from '../dto/transaction-meta.dto';
 import { ModelConstants } from '../models/model-constants';
-import { Tag } from '../models/tag.model';
 import { TransactionDto } from '../dto/transaction.dto';
+import { TransactionConverter } from '../utils/transaction-converter.utils';
 
 export class TransactionsService {
-  static async get(
+  static async getAll(
```

Comment:
This method does not return _all_ transactions, it returns _certain_ transactions based on the argument. Probably we should call it `getMany`.